wechatpy/work/client/api/invoice.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+# -*- coding: utf8 -*-
+
+# Date: 2020/05/06
+# Author: huimingz
+# Contact: huimingz12@outlook.com
+
+"""
+企业微信服务端电子发票API
+
+企业微信文档：https://work.weixin.qq.com/api/doc/90000/90135/90283
+"""
+
+from typing import Dict, List
+
+from wechatpy.client.api.base import BaseWeChatAPI
+
+# 发票初始状态，未锁定
+INVOICE_REIMBURSE_INIT = "INVOICE_REIMBURSE_INIT"
+# 发票已锁定，无法重复提交报销
+INVOICE_REIMBURSE_LOCK = "INVOICE_REIMBURSE_LOCK"
+# 发票已核销，从用户卡包中移除
+INVOICE_REIMBURSE_CLOSURE = "INVOICE_REIMBURSE_CLOSURE"
+
+invoice_reimburse_status_set = (INVOICE_REIMBURSE_INIT, INVOICE_REIMBURSE_LOCK, INVOICE_REIMBURSE_CLOSURE)
+
+
+class WeChatInvoice(BaseWeChatAPI):
+    """企业微信电子发票API"""
+
+    def get_info(self, card_id: str, encrypt_code: str) -> dict:
+        """查询电子发票
+
+        参考：https://work.weixin.qq.com/api/doc/90000/90135/90284
+
+        接口说明：报销方在获得用户选择的电子发票标识参数后，可以通过该接口查询电子发票的
+        结构化信息，并获取发票PDF文件。
+
+        **权限说明**：仅认证的企业微信账号有接口权限
+
+        返回结果参数说明请查看官方文档。
+
+        :param card_id: 发票id
+        :param encrypt_code: 加密code
+        :return: 发票信息
+        """
+        url = "card/invoice/reimburse/getinvoiceinfo"
+        data = {"card_id": card_id, "encrypt_code": encrypt_code}
+        return self._post(url, data=data)
```

Comment:
```suggestion
        return self._post("card/invoice/reimburse/getinvoiceinfo", data={"card_id": card_id, "encrypt_code": encrypt_code})
```

个人意见：其实这种中间变量没有用的话（而且以后大概率也不会有复杂逻辑），可以直接一行写完，更简洁

Reply:
（不过我这个 suggestion 是没经过 black 的，可能需要 black 一下）

Reply:
一行写完感觉不是太美观，这里我觉得还是保持这样吧